keyboards/epomaker/tide65/keymaps/default/keymap.c

```diff
@@ -0,0 +1,30 @@
+// Copyright 2024 SDK (@sdk66)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+// clang-format off
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [0] = LAYOUT_ortho_5x15( /* Base */
+        KC_ESC,   KC_1,       KC_2,       KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,       KC_MINS,  KC_EQL,   KC_BSPC,   KC_MUTE,
+        KC_TAB,   KC_Q,       KC_W,       KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,       KC_LBRC,  KC_RBRC,  KC_BSLS,   KC_DEL,
+        KC_CAPS,  KC_A,       KC_S,       KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,    KC_QUOT,            KC_ENT,    KC_PGUP,  
+        KC_LSFT,  KC_Z,       KC_X,       KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,               KC_SLSH,  KC_RSFT,  KC_UP,     KC_PGDN,
+        KC_LCTL,  KC_LGUI,    KC_LALT,    KC_SPC,   KC_SPC,   KC_SPC,   KC_SPC,                       KC_RALT,              MO(1),    KC_LEFT,  KC_DOWN,   KC_RGHT
+        ),  
+
+    [1] = LAYOUT_ortho_5x15( /* Base */
+        KC_GRV,   KC_F1,      KC_F2,      KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,     KC_F11,   KC_F12,   _______,   _______,
+        _______,  _______,    _______,    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,    KC_HOME,  KC_SCRL,  _______,   _______,
+        _______,  _______,    _______,    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,    KC_PSCR,            _______,   _______,
+        _______,  _______,    _______,    _______,  _______,  _______,  _______,  _______,  _______,  _______,              _______,  _______,  _______,   _______,
+        _______,  GU_TOGG,    _______,    EE_CLR,   EE_CLR,   EE_CLR,   EE_CLR,                       _______,              _______,  _______,  _______,   _______
+        )
+};
+
+void keyboard_pre_init_user(void) {
+    setPinOutput(A5);
+    writePinHigh(A5);
+}
```

Comment:
What is this for?

Reply:
This is to trigger the LED pins; only when the voltage level is high, the backlights will be turned on.

Reply:
This should be done at keyboard level then.